CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 #on the platform and configuration it is set to build in.
 project(tinyobjloader)
 cmake_minimum_required(VERSION 2.8.6)
```

Comment:
@syoyo What is the minimum version of CMake that you need to support? Does it need to remain 2.8.6?